### PR TITLE
WIP On Issue 68 - custom confirm modal (Issue Picked up by Kameron for Sprint 3- Moved to Separate PR)

### PR DESCRIPTION
**Pull request by cohen-j**
Started design of the custom confirm modal in new file, `confirmModal.js.` As I have only implemented the visual part of the modal (see below), it cannot be tested yet. The next thing I need to do for this issue is create functionality so it can be used in place of the browser's native `confirm()`.
image

### CODE REVIEW

**Comment by cohen-j** [comment]
a custom confirm modal (see below) should be implemented in all previous places where native browser confirm was used.

see `confirmModal.js` for the modal itself and `shared.js` for the helper function. other file changes pertain to either adding `<confirm-modal>` to html or changing `confirm()` to `modalConfirm()`

image

File: js/confirmModal.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            #confirm-message {
+                color: var(--color-primary);
+                font-weight: bold;
+            }
+        </style>
+
+        <div id="confirm-full">
+            <div id="confirm-dialog">
+                <div id="confirm-message-container">
+                    <h2 id="confirm-message"></h2>
```
Comment (AbhilashKotha) [inline js/confirmModal.js:50]: I don't think it is correct to use an h2 for a message. can you change it to a <p> with a bigger font size?

**Review by thehabes — CHANGES_REQUESTED** [review]
Tested on 
http://localhost:8080/ng.html#https://devstore.rerum.io/v1/id/6507cd02ee7021a5f6a454cf

I got the dialog
image

However, when I clicked 'OK', the delete did not happen.

Tested on http://localhost:8080/gloss-transcription.html#https://devstore.rerum.io/v1/id/65259f9ba2ad2e20e5172530

I got the dialog
image

However, when I clicked 'OK', the delete did not happen.

I do not believe this is functioning as expected.

**Review by cubap — CHANGES_REQUESTED** [review]
see @thehabes issue

**Comment by KamBustos** [comment]
For Sprint 3 I am picking up this issue that Josh was working on. So far, I have merged changes from the main, and I plan to refine and add to Josh's preexisting work for the remained of Sprint 3.

**Pull request closed without merging**